docs/Installation.md

```diff
@@ -140,6 +140,8 @@ Thanks for using SDT!
  _**NOTE:**_
 - Make sure the json file exists in the PDS data directory.
 
+Now to know how to update the `src/config/config.py` to reflect the new json files in the UI, follow steps mentioned in [Adding_new_distros](Adding_new_distros.md)
```

Comment:
So people know where to go, can you either mention Step 2 specifically, and/or link directly to that section of the doc? https://github.com/openmainframeproject/software-discovery-tool/blob/master/docs/Adding_new_distros.md#step-2-update-the-supported_distros-variable-in-configuration-file-pds_basesrcconfigconfigpy

Reply:
Sure okkk